Refresh.GameServer/Endpoints/ApiV3/DataTypes/Response/Users/ApiGameUserResponse.cs

```diff
@@ -52,6 +58,21 @@ public class ApiGameUserResponse : IApiResponse, IDataConvertableFrom<ApiGameUse
             Role = user.Role,
             Statistics = ApiGameUserStatisticsResponse.FromOld(user, dataContext)!,
             ActiveRoom = ApiGameRoomResponse.FromOld(dataContext.Match.RoomAccessor.GetRoomByUser(user), dataContext),
+            //TODO: this data should be cached
+            DiscordProfileInfo = user.DiscordProfileVisibility.Filter(
+                user,
+                dataContext,
+                ApiDiscordUserResponse.FromOld(dataContext.OAuth
+                    .GetOAuthClient<DiscordOAuthClient>(OAuthProvider.Discord)
+                    ?.GetUserInformation(dataContext.Database, dataContext.TimeProvider, user), dataContext)
+            ),
+            GitHubProfileInfo = user.GitHubProfileVisibility.Filter(
+                user,
+                dataContext,
+                ApiGitHubUserResponse.FromOld(dataContext.OAuth
+                    .GetOAuthClient<GitHubOAuthClient>(OAuthProvider.GitHub)
+                    ?.GetUserInformation(dataContext.Database, dataContext.TimeProvider, user), dataContext)
+            ),
```

Comment:
Is that making a request *every time a user is serialized?*

How do I put this...

# No.